XamlControlsGallery/Navigation/NavigationRootPage.xaml

```diff
@@ -71,6 +71,8 @@
             Header=" "
             HeaderTemplate="{StaticResource NavigationViewHeaderTemplate}"
             IsTabStop="False"
+            IsPaneOpen="False"
```

Comment:
You don't want to set this. When NavigationView's DisplayMode=Expanded, the Pane should be open by default. That's how you get this:
![image](https://user-images.githubusercontent.com/13142665/62992292-c6bf8700-be07-11e9-8f29-059d8f700631.png)

When IsPaneOpen="False" and DisplayMode=Expanded, you'll see this, which isn't how we want the app to start up:
![image](https://user-images.githubusercontent.com/13142665/62992352-ee165400-be07-11e9-9b52-03124ab193cf.png)

Reply:
Just double checking since this thread is still open - @YuliKl was your comment addressed?

Reply:
Yes. I'm happy with where the code changes ended up.